### PR TITLE
Take coverage of tests for Migemo/utf8

### PR DESCRIPTION
**Pull request by ichizok**
http://rhysd.hatenablog.com/entry/2017/11/27/091003

Vim 8.0.1372 (https://github.com/vim/vim/commit/ac112f01a6930c9d15cf0360b657373699916bfd) should have resolved covimerage error.

### CODE REVIEW

File: test/.themisrc [hunk 1]
```diff
@@ -1,15 +1,19 @@
 let g:repo_root = fnamemodify(expand('<sfile>'), ':h:h')
 
 call themis#option('exclude', g:repo_root . '/test/README.md')
-call themis#option('exclude', g:repo_root . '/test/Guardfile')
```
Comment (rhysd) [inline test/.themisrc]: The `Guardfile` is a local file in my environment. It is not committed to repo, but exists in my local directory. So please do not remove this line.

File: test/README.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
 ## How to execute tests
 
+It requires [vim-themis](https://github.com/thinca/vim-themis).
+
 ```console
-$ bundle install --path=.bundle
-$ rake test
+$ git clone https://github.com/thinca/vim-themis
+$ ./vim-themis/bin/themis .
```
Comment (rhysd) [inline test/README.md]: Please add `$ cd /path/to/clever-f.vim` line before `git clone` to clarify where the current working directory is.

File: test/README.md [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
 
-## How to watch file changes and run tests automatically
+## How to measure code coverage
+
+It requires [covimerage](https://github.com/Vimjas/covimerage).
 
 ```console
-$ rake guard
+$ pip install covimerage
+$ PROFILE_LOG=profile.txt ../vim-themis/bin/themis .
```
Comment (rhysd) [inline test/README.md]: Please add `$ cd /path/to/clever-f.vim/test` line before this line to clarify where the current working directory is.

**Comment by rhysd** [comment]
Thank you for this patch. This patch must improve running tests so I plan to merge this after above tiny review comments are fixed.

By this patch, on Vim 8.0.1372 or later, coverage for migemo feature can be taken. However, Vim on Travis CI is older. So I cannot confirm this patch works. Did you try this patch with the Vim after 8.0.1372 and confirmed it works?

**Comment by ichizok** [comment]
> By this patch, on Vim 8.0.1372 or later, coverage for migemo feature can be taken. However, Vim on Travis CI is older. So I cannot confirm this patch works. Did you try this patch with the Vim after 8.0.1372 and confirmed it works?

I confirmed only in my local. Would you like to add build-phase of latest Vim to CI (as with vital.vim) ?

**Comment by ichizok** [comment]
Homebrew macvim was updated to 8.0-143 (https://github.com/Homebrew/homebrew-core/commit/e84b6ac35e92aea912bf29ef2539483076f9973e) a few minutes ago.

**Comment by rhysd** [comment]
@ichizok Thank you for your fixes. Looks good. I'll check the result of merge commit to see if it works.

**Pull request merged**